src/pages/commits/TupleSelectWithRegexConditional/TupleSelectWithRegexConditional.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { action } from "@storybook/addon-actions";
+import { ProjectFilterOptions } from "types/commits";
+import TupleSelectWithRegexConditional from ".";
+
+export default {
+  title: "Pages/Commits/TupleSelectWithRegexConditional",
```

Comment:
rather than under the `Commits` folder, I feel like this might belong better under the `TupleSelect` folder as another variant 🤔